### PR TITLE
Figure of growth rate and corrected syntax errors in test.py

### PR DESCRIPTION
**Pull request by roxyboy**
Pushing a figure showing the analytical and numerical Eady growth rate and somehow the amendments in test.py were not reflected in the most recent merge so I corrected the syntax error again.

### CODE REVIEW

**Comment by rabernat** [comment]
I actually meant to just embed the image in your comment. Like this:

analytical_and_numerical_eadygrowthrate

**Comment by rabernat** [comment]
I can't merge your pull request, because, as you can see, it has conflicts with the master branch.

You need to [rebase your branch](https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request) before this can be merged..

**Comment by roxyboy** [comment]
I tried rebasing it following the instructions in link you sent me but I couldn't get it to work, so I just pulled the upstream repository (rabernat/oceanmodes.git) to my local repository. Since the amendments made before this pull request were implemented and I haven't made any further changes since then, is it possible to delete or close this pull request?

File: oceanmodes/test/test_modes.py [hunk 1]
```diff
@@ -238,12 +238,12 @@ def test_N2_const_equal_spacing(self):
         #########
         # Analytical solution for Eady growth rate
         #########
-        growthEady = np.zeros((1, len(k))
+        growthEady = np.zeros((len(k), 1))
         for i in range(len(k)):
-            if (np.tanh(.5*k)**-1 - .5*k) * (.5*k - np.tanh(.5*k)) < 0:
+            if (np.tanh(.5*k[i])**-1 - .5*k[i]) * (.5*k[i] - np.tanh(.5*k[i])) < 0:
```
Comment (rabernat) [inline oceanmodes/test/test_modes.py:243]: What about these typos? Don't they have to be corrected?

Comment (roxyboy) [inline oceanmodes/test/test_modes.py:243]: The current repository (rabernat/oceanmodes https://github.com/rabernat/oceanmodes/blob/master/oceanmodes/test/test_modes.py ) already has the code amended to:

```
for i in range(len(k)):
            if (np.tanh(.5*k[i])**-1 - .5*k[i]) * (.5*k[i] - np.tanh(.5*k[i])) < 0:
```

Comment (roxyboy) [inline oceanmodes/test/test_modes.py]: https://github.com/rabernat/oceanmodes/blob/master/oceanmodes/test/test_modes.py

**Comment by rabernat** [comment]
Ok, then if there is nothing useful in this PR, please close it.

**Pull request closed without merging**